modules/shared/components/molecules/UploadingImage/UploadingImage.module.css

```diff
@@ -3,22 +3,22 @@
 }
 
 .button {
-  @apply absolute top-4 right-4 w-8 h-8 rounded-full bg-white flex justify-center items-center outline-none focus:outline-none z-20;
+  @apply absolute top-4 right-4 z-40;
 }
 
 .layout {
-  @apply absolute w-full h-full top-0 left-0 bg-white opacity-95 flex justify-center items-center z-0;
+  @apply absolute w-full h-full top-0 left-0 bg-white opacity-95 flex justify-center items-center z-20;
 }
 
 .image-container {
-  @apply relative w-full h-full mb-1 overflow-hidden rounded-t-md;
+  @apply relative w-full h-full flex justify-center mb-1 overflow-hidden rounded-t-md;
 }
 
 .image {
-  @apply object-cover  w-full h-full;
+  @apply object-contain z-10;
 }
 .image-caption {
-  @apply relative z-50;
+  @apply relative z-20;
 }
 .image-caption input {
   @apply rounded-t-none;
```

Comment:
hope if you can make a storybook for that component so we can preview the design easily